README.md

```diff
@@ -81,7 +81,7 @@ utf_view has the following minimum compiler requirements:
 
 - GCC 14
 - Clang 19
-- MSVC 17.11.2 (earlier MSVC versions may also work but are not tested in CI)
+- MSVC 19.41
```

Comment:
This might actually work for your purposes.  I think the only failure was something to do with one of the range adaptors that I don't think you use.  I was essentially forced to up the supported MSVC version because they won't let me download any MSVC earlier thatn VS 2022.  So, fuck 'em.  No download, no support for your shitty compiler.

Anyway ..., you might be able to use the newest stl_interfaces without needing to up the minimum MSVC version.

Reply:
What's going on here isn't me upping the minimum MSVC version, it's just me fixing a bug in the readme file. The version I had here before was the Visual Studio version, not the MSVC version, so I changed it to the corresponding MSVC version as a drive-by fix.

![Screenshot 2024-10-24 at 6 51 55 PM](https://github.com/user-attachments/assets/54a8c4c6-ed03-4a7b-8153-a26795625f45)

Reply:
Aaah.  Gotcha.  I hate that there are three magic numbers for every release.